_data/examples.json

```diff
@@ -238,6 +238,10 @@
       "name": "layer_line_co2_concentration",
       "title": "Carbon Dioxide in the Atmosphere",
       "description": "Note&#58; When accessing aggregated argmax or argmin fields, the aggregated fields must be flattened using calculate (see https://github.com/vega/vega-lite/issues/3361)."
+    },
+    {
+      "name": "layer_line_multi_value_tooltip",
+      "title": "Multi-Value Tooltip",
```

Comment:
It would be nice to provide a `"description"` to provide a summary to help readers too. (You can use markdown in the "description" field.)

Reply:
Done!